package.json

```diff
@@ -53,6 +53,7 @@
     "babel-preset-react-hmre": "1.1.1",
     "babel-preset-stage-2": "6.24.1",
     "copy-webpack-plugin": "4.3.0",
+    "css-hot-loader": "^1.3.8",
```

Comment:
Can you remove the caret from the version? I keep all dependency versions to specific.

Reply:
done :)